tests/src/rules/no-duplicates.js

```diff
@@ -179,17 +187,24 @@ ruleTester.run('no-duplicates', rule, {
     }),
 
     test({
-      code: "import * as ns from './foo'; import {y} from './foo'",
-      // Autofix bail because first import is a namespace import.
```

Comment:
i'd prefer not to modify existing tests, to help ensure it's not a breaking change.

Reply:
Well, for this specific test, it IS a breaking change, because it currently is an error, which we don't want it to be anymore.

Reply:
turning an error into a non-error isn't breaking, it's a patch :-) i see that you've actually just moved this test up to "valid", so we're good here